packages/drafts-realm/package.json

```diff
@@ -4,10 +4,9 @@
   "main": "index.js",
   "license": "MIT",
   "devDependencies": {
-    "@cardstack/cardpay-sdk": "^1.0.53",
```

Comment:
I shall add this back in in another PR. Just want to unblock people who are attempting to build application to make cards